src/pydata_sphinx_theme/assets/styles/extensions/_ablog.scss

```diff
@@ -0,0 +1,51 @@
+/**
+ * ABlog
+ * ref: https://ablog.readthedocs.io/
+ */
+// HACK: ABlog has no CSS selector, but directly inserts elements into the sidebar
+// So we make an assumption here that any *top-level* sidebar items that are ul/h2/h3
+// Are attached to ABlog. This is hacky and we should try to get top-level ablog
+// CSS wrappers into ABlog instead.
+.bd-sidebar-primary .sidebar-start-items {
+  > h3,
+  > h2 {
+    margin-top: 1rem;
+  }
+
+  > ul {
+    list-style: none;
+    padding-left: 0;
+  }
+}
+
+// HACK: ABlog articles always seem to have the structure:
+// <section id="POST-TITLE-ID">{{ POST CONTENT }}
+// <div class="section">
+//   <div class="section"
+//      <span>previous button
+//      <span>empty space
+//      <span>next button
+//
+// So these rules make that assumption
+article.bd-article > section + div.section {
+  font-size: 1.2em;
+
+  span:first-child:before {
+    content: "\f104";
+    font-family: "Font Awesome 5 Free";
+    font-weight: 800;
+  }
+
+  span:last-child:after {
+    content: "\f105";
+    font-family: "Font Awesome 5 Free";
+    font-weight: 800;
+  }
+}
```

Comment:
What about overriding the template for the post nav? https://github.com/sunpy/ablog/blob/main/ablog/templates/postnavy.html

That could help avoiding the current hanging arrow at the ends of the post list and include next/previous and the arrow as part of the `<a>` like the rest of the theme.


<img width="768" alt="image" src="https://user-images.githubusercontent.com/1296209/178281400-c8de77af-3da8-460a-883d-1172ebd088d3.png">

Reply:
I think it's actually not that easy to over-ride templates with ABlog until this one is resolved:

- https://github.com/sunpy/ablog/issues/125